labels.md

```diff
@@ -0,0 +1,220 @@
+component/statistics            #80e58d     component/statistics                #d1fad7
+component/syncer                #ccbf0e     component/syncer                    #d1fad7
+component/test                  #6494f4     component/test                      #d1fad7
+component/tikv                  #8bef73     component/tikv                      #d1fad7
+component/tools                 #cbf232     component/tools                     #d1fad7
+component/transaction-need-LGT  #83e2d1     component/transaction (3lgtm)       #d1fad7
+component/usability             #0ad8bd     component/usability                 #d1fad7
+component/util                  #47e0b0     component/util                      #d1fad7
+contribution                    #b0ff5b     type/contributor ⭐️                 #1d76db
+errmsg                          #a7c938     type/error message                  #1d76db
+for new contributors            #c2e0c6     difficulty/Mentor                   #0e8a16
+help wanted                     #159818     status/help wanted                  #e6e6e6
+needs-cherry-pick-2.1           #000000     type/cherry pick 2.1                #1d76db
+needs-cherry-pick-3.0           #000000     type/cherry pick 3.0                #1d76db
+priority/P1                     #b60205     priority/low                        #eb6420
+priority/P2                     #FF7F50     priority/medium                     #0e8a16
+priority/non-release-blocker    #0052cc     priority/high                       #eb6420
+priority/release-blocker        #b60205     priority/blocker                    #eb6420
+proposal                        #eb6420     type/proposal                       #1d76db
+question                        #fbca04     type/question                       #1d76db
+release-note                    #1d76db     * <type/CHANGELOG>                    
+require-LGT3                    #d93f0b     type/require 3 reviews              #1d76db
+status/DNM                      #b60205     status/DNM                          #e6e6e6
+status/LGT1                     #d4c5f9     * <status/Waiting on review>            
+status/LGT2                     #5319e7     * <status/Waiting on review>            
+status/LGT3                     #330099     * <status/Waiting on review>            
+status/PTAL                     #2cbe4e     status/waiting on review            #e6e6e6
+status/ReqChange                #b60205     status/waiting on author            #e6e6e6
+status/TODO                     #207de5     *                                   
+status/WIP                      #fbca04     status/WIP                          #e6e6e6
+status/all tests passed         #2cbe4e     * <status/bot merge>                    
+status/can merge                #2cbe4e     status/bot merge                    #e6e6e6
+status/future                   #fbca04     ?                                   
+type/1.0 cherry-pick            #c2e0c6     *                                   
```

Comment:
By using labels like `type/cherry pick 2.1` to identify whether a PR on the master branch needs to be cherry-picked to release 2.1, we also need labels to identify which branch the PR is submitted on.

A typical use case is we need to check whether there still some un-merged PRs on a release branch, such as release-2.1, before releasing a new version on that release branch.

How about using the following labels to identify which branch the PR is submitted on:
```
branch/release-2.0
branch/release-2.1
branch/release-3.0
```